pkg/portal/portal.go

```diff
@@ -160,22 +161,36 @@ func (p *portal) setupRouter(kconfig *kubeconfig.Kubeconfig, prom *prometheus.Pr
 	allGroups := append([]string{}, p.groupIDs...)
 	allGroups = append(allGroups, p.elevatedGroupIDs...)
 
-	p.aad, err = middleware.NewAAD(p.log, p.audit, p.env, p.baseAccessLog, p.hostname, p.sessionKey, p.clientID, p.clientKey, p.clientCerts, allGroups, unauthenticatedRouter, p.verifier)
+	// we get the env var from within the function because it is not meant to be configurable
+	disableOauthOption := os.Getenv("DISABLE_OAUTH")
+	if disableOauthOption == "true" || env.IsCI() || env.IsLocalDevelopmentMode() {
+		p.baseAccessLog.Errorf("running in int, disableoauth=%s", disableOauthOption)
+		p.aad, err = middleware.NewIntAAD(p.elevatedGroupIDs, p.audit)
+	} else {
+		p.log.Error("not running in int")
```

Comment:
I don't think we need to log this when we're starting in regular mode, especially as error. I would just remove it.

Reply:
this is some leftover from debugging